test/blocks/marquee-anchors/marquee-anchors.test.js

```diff
@@ -0,0 +1,23 @@
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+const { default: init } = await import('../../../libs/blocks/marquee-anchors/marquee-anchors.js');
+describe('marquee-anchors', () => {
+  const marquees = document.querySelectorAll('.marquee-anchors');
+  marquees.forEach((marquee) => {
+    init(marquee);
```

Comment:
maybe in a `beforeEach` and have one `describe` block?

Reply:
@JasonHowellSlavin  - I updated the test file adding a beforeEach etc similar to other tests. 
I don't think this snippet however should be in that func. I also have just one describe block now. 
Anyhow I ran it,  and it checks out. LMK if this looks right? Thanks